pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UseDiamondOperator.xml

```diff
@@ -6,7 +6,7 @@
     <test-code>
         <description>Use Diamond</description>
         <expected-problems>2</expected-problems>
-        <expected-linenumbers>8,11</expected-linenumbers>
+        <expected-linenumbers>6,8</expected-linenumbers>
```

Comment:
interesting... I wonder why this didn't fail....

Reply:
The problem was, that the test class didn't have the suffix "Test" and that's why surefire didn't pick this up. So essentially the tests for this rule where never invoked during a maven build. I've just fixed that: 544238244beab6e6701df3675f362b3fb079582d